testnets/saagetestnet/assetlist.json

```diff
@@ -1,24 +1,25 @@
 {
     "$schema": "../../assetlist.schema.json",
-    "chain_id": "saage-internal-testnet-1.3",
+    "chain_id": "sit-5",
+    "chain_name": "saagetestnet5",
```

Comment:
can we make this match the folder name exactly? That's what we enforce on all the mainnet ones, so should probably have the same standard.